.editorconfig

```diff
@@ -4,6 +4,6 @@ root = true
 end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
-insert_final_newline = true
```

Comment:
Because of a bug in the VScode that disallows overriding this value for only certain file extensions, which is what I wanted (.Dockerfile in this case).